app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,26 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run, initiator, benchmark_type, benchmark_result_type, category)
+
+    current_initiator = benchmark_run.initiator_id
+    last_initiator = BenchmarkRun.where(initiator_type: Commit, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).order(:created_at).first.initiator_id
+
+    previous_benchmark_runs = BenchmarkRun.where(initiator_type: Commit, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).offset(last_initiator - current_initiator).limit(1000)
+
+    commit_results = Array.new
+    previous_benchmark_runs.drop(1).each do |run|
+      commit_results << run.result[category].to_f
+    end
+
+    results_avg = commit_results.inject{ |sum, el| sum + el }.to_f / commit_results.size
+    results_standard_deviation = Math.sqrt(commit_results.inject(0){|accum, i| accum +(i-results_avg)**2 }/(commit_results.length - 1).to_f)
+    results_param = results_avg + 2*results_standard_deviation
+
+    if benchmark_run.result[category].to_f > results_param
+      `curl --user "RubyBenchBot:a12345a" -X POST --data '{"title":"Benchmark Regression Detected"}' https://api.github.com/repos/ruby-bench/ruby-bench-suite/issues`
```

Comment:
Please look up how HTTP calls are usually made in Ruby. We usually don't do it via a system call to `curl`